ecs-cli/modules/clients/aws/secretsmanager/client.go

```diff
@@ -90,3 +97,63 @@ func (c *secretsManagerClient) PutSecretValue(input secretsmanager.PutSecretValu
 
 	return output, nil
 }
+
+func (c *secretsManagerClient) GetSecretValue(secretName string) (string, error) {
+	input := &secretsmanager.GetSecretValueInput{
+		SecretId:     aws.String(secretName),
+		VersionStage: aws.String("AWSCURRENT"), // VersionStage defaults to AWSCURRENT if unspecified
```

Comment:
nit: Can we add a comment linking to https://docs.aws.amazon.com/secretsmanager/latest/apireference/API_GetSecretValue.html#API_GetSecretValue_RequestSyntax

Reply:
Anything in particular that's confusing about this? We don't do this for other wrapped API methods on clients, but happy to make this clearer somehow.

Reply:
No, nothing is confusing. I just had to look it up online to see if they had an explanation for "AWSCURRENT". I like providing a `//See <link>`, but it's a nitpick.

Reply:
Given that it's the [default value](https://docs.aws.amazon.com/secretsmanager/latest/apireference/API_GetSecretValue.html#SecretsManager-GetSecretValue-request-VersionId), I'm guessing we can probably remove this entirely? I lifted a lot of this code from the Secrets Manager console but i'm wondering now if it's even necessary.